appveyor.yml

```diff
@@ -0,0 +1,128 @@
+environment:
+
+  matrix:
+    # See: http://www.appveyor.com/docs/installed-software#python
+
```

Comment:
This matrix section can be improved by removing all the duplication, see: https://packaging.python.org/guides/supporting-windows-using-appveyor/

Reply:
I *think* you can set Python version outside matrix for all matrix jobs e.g. 

```
environment:
  PYTHON_VERSION: "2.7"
  matrix:
    TOXENV: pydef
...
```

Reply:
for now sure since we are only python 2 compatible but in the future adding 3.x python testing it would have to be non-global. changing now later it doesn't make a difference too much.